gtfs-realtime/spec/en/reference.md

```diff
@@ -439,14 +439,14 @@ TripDescriptor.route_id cannot be used within an Alert EntitySelector to specify
 
 #### Fields
 
-| _**Field Name**_ | _**Type**_ | _**Required**_ | _**Cardinality**_ | _**Description**_ |
```

Comment:
Are there any changes in this section? If it is just moved, can we do that in separate change?